framework/src/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -2081,6 +2082,14 @@ public AnnotatedTypeMirror getErased() {
         }
     }
 
+    /**
+     * The primary annotation on an intersection type is the GLB of the primary annotations
+     * of the bounds.
```

Comment:
Also say that there is no syntax to explicitly write the primary annotations in source code.